src/labels_transfer/api/common_arguments_2.yaml

```diff
@@ -0,0 +1,151 @@
+                        description: "The target labels to transfer. Override using the `--reference_obs_targets` argument."
+                        required: true
+        - name: "--reference_obsm_features"
+          type: string
+          required: false
+          description: |
+            The `.obsm` key of the embedding to use for the classifier's training. If not provided, the `.X` slot will be used instead.
+            Make sure that embedding was obtained in the same way as the query embedding (e.g. by the same model or preprocessing).
+          example: X_scvi
+        - name: "--reference_obs_targets"
+          type: string
+          default: [ ann_level_1, ann_level_2, ann_level_3, ann_level_4, ann_level_5, ann_finest_level ]
+          required: false
+          multiple: true
+          description: The `.obs` key(s) of the target labels to tranfer.
+
+    - name: Outputs
+      arguments:
+        - name: "--output"
+          type: "file"
+          direction: output
+          required: true
+          description: "The query data in .h5mu format with predicted labels transfered from the reference."
+          info:
+            label: "Output data"
+            file_format:
+              type: h5mu
+              mod:
+                rna:
+                  description: "Modality in AnnData format containing RNA data."
+                  required: true
+                  obs:
+                    - type: "string"
+                      name: "predictions"
+                      description: "The predicted labels. Override using the `--output_obs_predictions` argument."
+                      required: true
+                    - type: "double"
+                      name: "uncertainty"
+                      description: "The uncertainty of the predicted labels. Override using the `--output_obs_uncertainty` argument."
+                      required: false
+                  obsm:
+                    - type: "double"
+                      name: "X_scvi"
+                      description: 
+                        The embedding used for the classifier's inference. Could have any name,
+                        specified by `input_obsm_features` argument."
+                      required: false
+        - name: "--output_obs_predictions"
+          type: string
+          example: 
+          required: false
+          multiple: true
+          description: |
+            In which `.obs` slots to store the predicted information.
+            If provided, must have the same length as `--reference_obs_targets`.
+            If empty, will default to the `reference_obs_targets` combined with the `"_pred"` suffix.
+        - name: "--output_obs_probability"
+          type: string
+          example: 
+          required: false
+          multiple: true
+          description: |
+            In which `.obs` slots to store the uncertainty of the predictions.
```

Comment:
It should be "probability" instead of "uncertainty". Also, I would suggest keeping it consistent (i.e. either output probabilities or uncertainties in all the components)

Reply:
makes sense!